Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -292,11 +297,34 @@ class ButtonClickListener : Object, IOnClickListener
 
 			#endregion
 
-			public void OnClick(global::Android.Views.View v)
+			public void OnClick(AView v)
 			{
 				var renderer = v.Tag as ButtonRenderer;
 				((IButtonController)renderer?.Element)?.SendClicked();
 			}
 		}
+
+		class ButtonTouchListener : Object, IOnTouchListener
+		{
+			#region Statics
+
+			public static readonly Lazy<ButtonTouchListener> Instance = new Lazy<ButtonTouchListener>(() => new ButtonTouchListener());
+
+			#endregion
```

Comment:
I haven't looked at history. Maybe the regions were left from a previous version. They serve no purpose in this case. Normally, I'd refactor them out, but it's between you and me, I do a lot of refactoring sometimes. :) If you are adding new code, I don't think the region needs to be there.